### DISCUSSION TITLE
[General] <How can we list our DEX on zkSync Ecosystem?>

### DISCUSSION THREAD

**Discussion by ZkSwapFinance** [zkSync-Community-Hub/zksync-developers#96] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

Dear zkSync Team,

I'm Ryan, representing zkSwap Finance (https://twitter.com/zkSwap_finance). Over the past 6 months, we've been deeply involved with zkSync, from building a community to developing the project from its inception, testnet, and mainnet phases. Despite our efforts, we've faced challenges in applying for listing within the zkSync ecosystem. Regrettably, we have not received any feedback or conditions for listing.

As dedicated contributors to this project, early adopters of zkSync, and active members of the community surrounding it, it's disheartening to find ourselves in this situation. We genuinely hope that the zkSync team or Matter Labs will take a moment to read our message and offer a response.

Feel free to contact me on Telegram: https://t.me/zkSwap_Ryan

Best regards,
Ryan

### What is the specific issue or error you're encountering?

Listing

### Can you share the error messages or logs you're receiving, if any?

Listing

### Have you made any recent changes to the contract before encountering this issue?

Listing

### Are there any external libraries or contracts that your contract interacts with?

Listing

### Can you provide the relevant portions of your contract code where the issue is occurring?

Listing

### Have you tried to isolate the problem, and if so, what were the results?

Listing

### What steps have you already taken to try to resolve the issue?

Listing

**Comment by uF4No** [comment, marked as answer]
Hey @ZkSwapFinance You're project is under review and it'll be added to the ecosystem page soon

**Reply by ZkSwapFinance** [reply]
Dear @uF4No,

Thank you for your response. We genuinely aspire for recognition from zkSync, as it significantly motivates us.

Could I know the estimated timeframe? I've been aware that our submission is under review for quite some time, but we haven't received any feedback for several months now.

Best regards,
Ryan

**Comment by vatosherrod** [comment]
As an AI language model, I'm unable to directly facilitate listing requests or address issues related to specific projects within the zkSync ecosystem. However, I recommend reaching out directly to the zkSync team or Matter Labs via their official communication channels, such as email or Discord, to inquire about the listing process and discuss any potential requirements or conditions. They will be in the best position to provide guidance and assistance regarding your listing request.
